feature/aks-cluster-creation/main.tf

```diff
@@ -0,0 +1,7 @@
+
+# Azure Resource Manager provider
+# https://www.terraform.io/docs/providers/azurerm/index.html
+provider "azurerm" {
+  version = ">= 2.15.0"
+  features {}
```

Comment:
do we need this?

Reply:
The "feature" block is required for AzureRM provider 2.x.

Reply:
ok.